Documentation.app/Contents/MacOS/Quick start.playground/Pages/Creating a Playground Book.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,23 @@
+// nef:begin:header
+/*
+ layout: docs
+ */
+// nef:end
+
+/*:
+Swift Playgrounds is a revolutionary app that makes possible develop on iPad. In the latest updates, Swift Playgrounds 3.x has added a new feature [UserModules](https://developer.apple.com/documentation/swift_playgrounds/structuring_content_for_swift_playgrounds/using_modules_to_share_code_in_a_playground_book); it let you include swift code and make it available across multiple chapters like modules.
+
+nef takes advantage of these new possibilities and advancements in Swift Package Manager to build a Playground Book with external dependencies from a Swift Package specification.
+
+```bash
+➜ nef ipad --name PlaygroundName --package Package.swift --output ~/Desktop
+```
+ 
+ It will create a Playground Book (`PlaygroundName`) with support for the external dependencies defined in `Package.swift` and save it in `~/Desktop`
```

Comment:
Perhaps add the content of a sample `Package.swift` for people to know what this looks like. People may be used to adding dependencies through Xcode and never had to write a manifest file.

Reply:
Great!